pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2003,6 +2112,17 @@ func buildThriftProxy(thriftOpts *thriftListenerOpts) *thrift.ThriftProxy {
 	}
 }
 
+func buildDubboProxy(dubboOpts *dubboListenerOpts) *dubbo.DubboProxy {
```

Comment:
move to networkfilters.go.

Reply:
Unlike mysql/mangodb, Dubbo filter handles the routing by itself instead of passing it through to a tcp proxy filter.